### ISSUE TITLE
Include Pluto horizon calculator

### ISSUE THREAD

**Issue by jhidding** [bvreede/horizon-spotting#4]
I would like to compute the horizon distance on Pluto.

Could we have a selection menu for the planet you're on?

People need to know.

**Comment by samnooij** [comment]
Yes, great idea! I was thinking of downloading and/or parsing wikipedia to get the numbers into Python. (So that is related to #1: don't write numbers manually in markdown, but have the computer get and print them for you.)
Perhaps not an easy change, but would be nice if we can pull this of, don't you think?
